### PR TITLE
RED-26835 Add Geo support in CRDB 5.4.2

### PR DESCRIPTION
**Pull request by bmansheim**

### CODE REVIEW

**Comment by bmansheim** [comment]
@Paz-Yanover We're waiting for a response from Yuval about whether we need to add more information about how to use GEO.

**Comment by bmansheim** [comment]
@yuval@redislabs.com Can you review these changes please?

File: content/rs/developing/crdbs/_index.md [hunk 1]
```diff
@@ -108,21 +108,22 @@ requirements in CRDBs.
...
-|  Hashes | Supported. Hash fields are treated as strings or counters. [See detailed information]({{< relref "/rs/developing/crdbs/developing-hashes-crdb.md" >}}) |
-|  Integer Counters | Supported, [see detailed information]({{< relref "/rs/developing/crdbs/strings.md#counters" >}}) |
-|  Lists | Supported, [See detailed information]({{< relref "/rs/developing/crdbs/developing-lists-crdb.md" >}}) |
-|  Sets | Supported, [see detailed information]({{< relref "/rs/developing/crdbs/developing-sets-crdb.md" >}}) |
-|  Strings | Supported, [see detailed information]({{< relref "/rs/developing/crdbs/strings.md" >}}) |
-|  Sorted Sets | Supported, [See detailed information]({{< relref "/rs/developing/crdbs/developing-sorted-sets-crdb.md" >}}) |
-|  Bitsets | Not currently supported |
-|  Streams | Not currently supported |
+| Float Counters | [Supported]({{< relref "/rs/developing/crdbs/strings.md#counters" >}}) |
+| Geo | [Supported]({{< relref "/rs/developing/crdbs/developing-sorted-sets-crdb.md" >}}) |
```
Comment (Paz-Yanover) [inline content/rs/developing/crdbs/_index.md]: Geospatial

File: content/rs/developing/crdbs/developing-sorted-sets-crdb.md [hunk 2]
```diff
@@ -5,6 +5,10 @@ weight: $weight
 alwaysopen: false
 categories: ["RS"]
 ---
+{{% note %}}
+[Redis Geo](https://redis.io/commands/GEOADD) is based on Sorted Sets, so the same CRDB development instructions apply to Geo.
```
Comment (Paz-Yanover) [inline content/rs/developing/crdbs/developing-sorted-sets-crdb.md]: Geospatial

**Pull request merged**